server/integration_tests/test_data/fulfillment_api_correlation/chart_config.json

```diff
@@ -814,48 +814,7 @@
         ]
       }
     ],
-    "exploreMore": {
```

Comment:
the dropped svs are commute related (dropped because there is a limit to number of svgs that get opened and new commute data has caused these svs to not be seen)